wagtailio/static/sass/components/_logo-card.scss

```diff
@@ -34,9 +37,29 @@
         margin-top: auto;
     }
 
-    &:is(a):hover {
-        #{$root}__heading {
-            color: $color--teal;
+    &:is(a) {
```

Comment:
**Praise:** Love the use of the futuristic pseudo selector here. 

<img src="https://media.giphy.com/media/T0H8OTMGl7ZzW/giphy.gif" alt="Robot swinging arms"/>

Reply:
😄 I can see myself using this a lot more going forward!